retis/src/core/probe/manager.rs

```diff
@@ -547,7 +561,17 @@ impl ProbeRuntimeManager {
 
     /// Get the list of all currently attached probes.
     pub(crate) fn attached_probes(&self) -> Vec<String> {
-        self.probes.clone().into_iter().collect()
+        self.probes.keys().cloned().collect()
+    }
+
+    /// Get the list of all current global options.
+    pub(crate) fn global_options(&self) -> Vec<ProbeOption> {
+        self.global_probes_options.clone().into_iter().collect()
+    }
```

Comment:
This is added in patch 1 and then the only user is removed.

Reply:
argh, I squashed the wrong patches